WordPressAuthenticator/Analytics/AuthenticatorAnalyticsTracker.swift

```diff
@@ -222,18 +226,28 @@ public class AuthenticatorAnalyticsTracker {
     /// Shared Instance.
     ///
     public static var shared: AuthenticatorAnalyticsTracker = {
-        let configuration = AuthenticatorAnalyticsTracker.Configuration(
-            appleEnabled: WordPressAuthenticator.shared.configuration.enableUnifiedApple,
-            googleEnabled: WordPressAuthenticator.shared.configuration.enableUnifiedGoogle,
-            siteAuthenticationEnabled: WordPressAuthenticator.shared.configuration.enableUnifiedSiteAddress)
-        
-        return AuthenticatorAnalyticsTracker(configuration: configuration)
+        return AuthenticatorAnalyticsTracker(configuration: defaultConfiguration())
     }()
     
     struct Configuration {
         let appleEnabled: Bool
         let googleEnabled: Bool
-        let siteAuthenticationEnabled: Bool
+        let siteAddressEnabled: Bool
+        let wpComEnabled: Bool
+    }
+    
+    private class func defaultConfiguration() -> Configuration{
+        // When unit testing, WordPressAuthenticator is not always initialized.
+        // The following code ensures we have configuration defaults even if that's the case.
+        guard WordPressAuthenticator.isInitialized() else {
+            return Configuration(appleEnabled: false, googleEnabled: false, siteAddressEnabled: false, wpComEnabled: false)
+        }
+        
+        return Configuration(
+            appleEnabled: false,
```

Comment:
Shouldn't this check the feature flag? (`WordPressAuthenticator.shared.configuration.enableUnifiedApple`)

Reply:
Not really.  The reason being that the feature flag is enabled for developers but I still have to develop and test the logging for SIWA.

Reply:
Oh, but to further clarify, this will changed to follow the feature flag when I push the logging for SIWA.

Reply:
Ah, gotcha. Thanks for the explanation.